Sockets/BackEndMonitoringSockets/src/CAcceptor.cpp

```diff
@@ -40,32 +40,30 @@ bool CAcceptor::Initialize(const std::string& ip_address,
 	return m_is_acceptor_initialized;
 }
 
-bool CAcceptor::Accept(int& connected_socket_fd)
+bool CAcceptor::AcceptNewClient(int& socket_fd)
 {
-	connected_socket_fd = c_error_socket;
+	int connected_socket_fd = c_error_socket;
```

Comment:
We do not need this variable in this function

Reply:
Deleted